.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   test:
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Can you borrow from the validation plugin the dynamic job name when experimental is true.

Reply:
`experimental` isn't used in the infrastructure repo (which makes sense, because infrastructure tests the versions we'll actually use in production). `fail-fast` is something else, determining whether all jobs should give up as soon as one fails.